File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 03/Lab3JAVA/AthleticsCompetition/src/domain/Enrollment.java
package domain;

public class Enrollment {
    Integer idChild;
    Integer idRound;

    /**
     * constructor for enrollment
     * @param idChild - integer, id of the child
     * @param idRound - integer, id of the round
     */
    public Enrollment(Integer idChild, Integer idRound) {
        this.idChild = idChild;
        this.idRound = idRound;
    }

    /**
     * getter for child id
     * @return - integer
     */
    public Integer getIdChild() {
        return idChild;
    }

    /**
     * setter for child id
     * @param idChild - integer
     */
    public void setIdChild(Integer idChild) {
        this.idChild = idChild;
    }

    /**
     * getter for round id
     * @return - integer
     */
    public Integer getIdRound() {
        return idRound;
    }

    /**
     * setter for round id
     * @param idRound - integer
     */
    public void setIdRound(Integer idRound) {
        this.idRound = idRound;
    }
}

File: UBB/Fourth Semester/Projecting and Programming Methods/Laboratory 03/Lab3C#/AthleticsCompetition/domain/Enrollment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AthleticsCompetition.domain
{
    class Enrollment
    {
        /// <summary>
        /// constructor for enrollment
        /// </summary>
        /// <param name="idChild"> integer </param>
        /// <param name="idRound"> integer </param>
        public Enrollment(int idChild, int idRound)
        {
            this.IdChild = idChild;
            this.IdRound = idRound;
        }

        /// <summary>
        /// getter and setter for child id
        /// </summary>
        protected int IdChild { get; set; }

        /// <summary>
        /// getter and setter for round id
        /// </summary>
        protected int IdRound { get; set; }

    }
}
